remote-continuous-query/src/main/java/org/infinispan/tutorial/simple/remote/query/InstaPost.java

```diff
@@ -0,0 +1,45 @@
+package org.infinispan.tutorial.simple.remote.query;
+
+import org.infinispan.protostream.annotations.ProtoDoc;
+import org.infinispan.protostream.annotations.ProtoField;
+
+/**
+ * This class is annotated with the infinispan Protostream support annotations. With this method, you don't need to
+ * define a protobuf file and a marshaller for the object. See this tutorial for additional explanations:
+ * <p>
+ * https://blog.infinispan.org/2018/06/making-java-objects-queryable-by.html
+ */
+@ProtoDoc("@Indexed")
+public final class InstaPost {
+
+   @ProtoDoc("@Field")
+   @ProtoField(number = 1, required = true)
```

Comment:
Required fields are going away in the future. They are still supported ATM of course but it would be ideal not to use them in tutorials to make them future-proof.

Reply:
how should I do it then ?

Reply:
if I remove required I get this error
Exception in thread "main" ```org.infinispan.protostream.annotations.ProtoSchemaBuilderException: Primitive field 'bornYear' of class org.infinispan.tutorial.simple.remote.query.Person should be marked required or should have a default value.```

I have to provide a default value instead?